test/stream-serializers.test.js

```diff
@@ -1,18 +1,17 @@
 'use strict'
 
-const t = require('tap')
-const test = t.test
+const { test } = require('node:test')
 const Fastify = require('..')
 const Reply = require('../lib/reply')
 
-test('should serialize reply when response stream is ended', t => {
+test('should serialize reply when response stream is ended', (t, done) => {
   t.plan(3)
   const stream = require('node:stream')
   const fastify = Fastify({
     logger: {
       serializers: {
         res (reply) {
-          t.type(reply, Reply)
+          t.assert.ok(reply, Reply)
```

Comment:
i think we should keep the test to ensure its the proper type here

Reply:
Yeah in any case ok is different